packages/lib-grommet-theme/src/index.js

```diff
@@ -186,7 +186,10 @@ const theme = deepFreeze({
           unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
         }
       `
-    }
+    },
+    input: {
+      weight: 'inherit'
```

Comment:
Just noticed this here... Would you update its changelog and note this change? It can be listed as unpublished for now until we're read to publish another update.

Reply:
My bad, I forgot that the theme is published. Will unpublished changes be picked up when we deploy master? I can't remember if the classifier installs dependencies from npm or symlinks them from the local file system.

Reply:
I think it's using symlinks if the version number in the package.json matches the version number in the theme's package.json, which still should be true. We can confirm on merge with staging.